cmon/client.go

```diff
@@ -143,6 +144,41 @@ func (client *Client) RequestBytes(module string, reqBytes []byte, noAutoAuth ..
 	return resBytes, nil
 }
 
+func (client *Client) GetReverseProxy(target string, headers http.Header) (*httputil.ReverseProxy, error) {
+
+	targetURL, err := url.Parse(target)
+	if err != nil {
+		return nil, fmt.Errorf("Error parsing target URL")
+	}
+
+	if client.ses != nil {
+		headers.Set("cookie", client.ses.String())
```

Comment:
it might be better to use https://pkg.go.dev/net/http#Request.AddCookie

Reply:
eg. https://github.com/severalnines/cmon-proxy/pull/27

Reply:
i've just tried it, but it does not work for me, i suspect because we need to replace cmon-sid cookie, and if browser already has one it fail authentication because wrong cookie is being checked in cmon-ssh